### PR TITLE
[monix] scaladocs

### PR DESCRIPTION
**Pull request by fwbrasil**

### CODE REVIEW

File: kyo-stm/shared/src/main/scala/kyo/TRef.scala [hunk 1]
```diff
@@ -6,11 +6,6 @@ import scala.annotation.tailrec
 
 /** A transactional reference that can be modified within STM transactions. Provides atomic read and write operations with strong
   * consistency guarantees.
-  *
```
Comment (fwbrasil) [inline kyo-stm/shared/src/main/scala/kyo/TRef.scala:9]: minor unrelated scaladoc fix

**Review by hearnadam — APPROVED** [review]
